packages/main/src/SplitButton.ts

```diff
@@ -372,8 +361,7 @@ class SplitButton extends UI5Element {
 	}
 
 	_arrowButtonPress(e: MouseEvent) {
-		e.preventDefault();
-		this.arrowButton!.focus();
+		e.stopPropagation();
```

Comment:
we're also removing the preventDefaut, are we sure that we want that?

Reply:
I think it was added to force the focus to be on the arrow button, but since focus is managed differently now it's no longer necessary.

Reply:
Not a good idea in general to prevent the default behavior on button press. If used in a form the default behavior would be to trigger form submit.